test/integration/node-middleware.test.ts

```diff
@@ -0,0 +1,90 @@
+import { createServer } from "http";
+
+import fetch from "node-fetch";
+
+import { Webhooks, createNodeMiddleware, sign } from "../../src";
+import { EmitterWebhookEvent } from "../../src/types";
+import { pushEventPayload } from "../fixtures";
+
+const signatureSha256 = sign(
+  { secret: "mySecret", algorithm: "sha256" },
+  JSON.stringify(pushEventPayload)
+);
+
+describe("createNodeMiddleware(webhooks)", () => {
+  it("README example", async () => {
+    const webhooks = new Webhooks<EmitterWebhookEvent>({
```

Comment:
```suggestion
  it("README example", async () => {
    expect.assertions(3);
  
    const webhooks = new Webhooks<EmitterWebhookEvent>({
```

as you've got an expect in an event (`webhooks.on("push", (event) => {`)